runfolder/services.py

```diff
@@ -162,8 +162,19 @@ def get_runfolder_state(self, runfolder):
         If the file .arteria/state exists, it will determine the state. If it doesn't
         exist, the existence of the marker file RTAComplete.txt determines the state.
         """
-        completed_marker_files = self._configuration_svc["completed_marker_files"]
-        completed_grace_minutes = self._configuration_svc["completed_marker_grace_minutes"]
+        completed_marker_files = None
+        try:
+            completed_marker_files = self._configuration_svc["completed_marker_file"]
+            if isinstance(completed_marker_files, basestring):
+                completed_marker_files = [completed_marker_files]
+        except KeyError:
+            pass
```

Comment:
cool, why not raise the ConfigurationError from line 183 here instead?

Reply:
Oh, you're right, I should move that up